### PR TITLE
implemented interceptor-requester debug handler

### PR DESCRIPTION
**Pull request by iulianpascalau**
implemented interceptor-requester debug handler

### CODE REVIEW

File: cmd/node/config/config.toml [hunk 1]
```diff
@@ -366,3 +366,7 @@
 [BlockSizeThrottleConfig]
     MinSizeInBytes = 943718 # 943718 is 90% from 1MB or 104857 is 10% from 1MB
     MaxSizeInBytes = 943718 # 943718 is 90% from 1MB
+
+[Debug]
+    Enabled = true
+    CachersSize = 10000
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/node/config/config.toml:372]: added

File: api/node/routes.go [hunk 2]
```diff
@@ -39,19 +43,26 @@ type shardStatisticsResponse struct {
 	AverageTPS            *big.Int `json:"averageTPS"`
 	PeakTPS               float64  `json:"peakTPS"`
 	CurrentBlockNonce     uint64   `json:"currentBlockNonce"`
-	TotalProcessedTxCount *big.Int `json:"totalProcessedTxCount"`
+	TotalProcessedTxCount *big.Int `json:"totalProce	assert.Equal(t, queryResponse.Error, apiErrors.ErrInvalidAppContext.Error())ssedTxCount"`
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply api/node/routes.go:46]: Wondering how that could have happened :)

File: debug/resolver/interceptorResolver.go [hunk 3]
```diff
@@ -2,13 +2,28 @@ package resolver
...
+	"time"
 
 	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/debug"
 	"github.com/ElrondNetwork/elrond-go/storage/lrucache"
 )
 
-type request struct {
+const requestEvent = "request"
```
Comment (BeniaminDrasovean) [inline debug/resolver/interceptorResolver.go:16]: Could use a single const keyword with declarations between parantheses.

**Comment by iulianpascalau** [comment]
This needs reworking in the area of injecting the new debug component

**Pull request closed without merging**